File: Buoi1/Big-O/O(n).py
# Vòng lặp đơn trong Python

def print_number(n):
    for i in range(1, n + 1):
        print(i, end=" ")  # In trên cùng một dòng với khoảng trắng
    print()  # Xuống dòng sau khi hoàn thành vòng lặp

def main():
    n = 5
    print_number(n)  # Output: 1 2 3 4 5

if __name__ == "__main__":
    main()

File: Buoi1/Big-O/O(n).jl
# Vòng lặp đơn trong Julia

function printNumber(n)
    for i in 1:n
        print("$i ")
    end
    println()  # In xuống dòng sau khi hoàn thành vòng lặp
end

function main()
    n = 5
    printNumber(n)  # Output: 1 2 3 4 5
end

main()
